src/pages/workspace/tags/WorkspaceViewTagsPage.tsx

```diff
@@ -234,6 +235,17 @@ function WorkspaceViewTagsPage({route}: WorkspaceViewTagsProps) {
                     cancelText={translate('common.cancel')}
                     danger
                 />
+                <View style={[styles.pv4, styles.ph5]}>
+                    <ToggleSettingOptionRow
+                        title={translate('common.required')}
+                        switchAccessibilityLabel={translate('common.required')}
+                        isActive={Boolean(policyTagList?.required)}
+                        onToggle={(on) => Tag.setPolicyTagsRequired(policyID, on, route.params.orderWeight ?? 0)}
+                        pendingAction={currentPolicyTag.pendingFields?.required}
+                        errors={currentPolicyTag.errors}
```

Comment:
We need `errorFields` to not mix errors, we have 2 errors sources, errors from changing the tag name and errors from changing the tag required status

Reply:
what do you think we should do here? 🤔

Reply:
Use `errorFields` i.e. instead of storying the error in `currentPolicyTag.errors` store it in `currentPolicyTag.errorFields.required` (update the `setPolicyTagsRequired` function)

Also if the BE is sending the error for the API request make sure it's sending it under `errorFields.required`

Reply:
I think there is no specific error set on the backend side, so I'm setting generic error [here](https://github.com/Expensify/App/pull/42972/commits/3c18dd66da42f3bab2b67fa82fbe7dac6ffcc00d)